Performance-Tracing.md

```diff
@@ -0,0 +1,156 @@
+# Performance Tracing
+
+TypeScript 4.1 introduced a `generateTrace` flag to make it easier to identify the parts of a program that are taking the most time to compile.
+
+## Warnings
+
+This feature is still experimental and will likely change significantly in TS 4.2.
+In particular, though it would be easy and helpful to post-process the trace files, you should assume that your post-processing tools will be broken by TS 4.2.
```

Comment:
```suggestion
In particular, though it would be easy and helpful to post-process the trace files, you should assume that any post-processing tools that may be built will be broken by TS 4.2.
```

Reply:
"your" -> "any such"